src/fixpoint_sdk/lib/requests.py

```diff
@@ -48,18 +48,19 @@ def create_openai_input_log(
         model_name: str,
         request: types.OpenAILLMInputLog,
         trace_id: typing.Optional[str] = None,
+        mode: types.ModeType = types.ModeType.MODE_UNSPECIFIED,
     ) -> types.InputLog:
         """Create an input log for an LLM inference request."""
         url = f"{self.base_url}/v1/openai_chats/{model_name}/input_logs"
-
         input_log_req = types.CreateLLMInputLogRequest(
             model_name=model_name,
             messages=request["messages"],
             user_id=request.get("user", None),
             temperature=request.get("temperature", None),
             trace_id=trace_id,
+            mode=mode,
         )
-
+        self._post_to_fixpoint(url, input_log_req.to_dict())
```

Comment:
This line seems like it is a bug? We already post to fixpoint below and return the value.